SoftLayer/managers/cdn.py

```diff
@@ -141,7 +141,7 @@ def purge_content(self, unique_id, path):
         :returns: A Container_Network_CdnMarketplace_Configuration_Cache_Purge array object
         """
 
-        return self.cdn_purge.createPurge(unique_id, path)
+        return self.cdn_purge.createPurge(unique_id, "/" + path)
```

Comment:
I think in future is better start to implement with 'string formatting' to join strings